### PR TITLE
Add missing npm dependency to the install_dependencies

### PR DESCRIPTION
**Pull request by jkonecny12**
This will enable us to use toolbox containers pretty easily for the new UI
development.

Just do:

toolbox create anaconda
toolbox enter anaconda
sudo ./scripts/testing/install_dependencies.sh

Then you have everything you need.

### CODE REVIEW

**Review by VladimirSlavik — CHANGES_REQUESTED** [review]
- I think `npm` is needed to build, so it should go into the spec.
- `make` also seems suspicious to me. How come it's not pulled in via autotools?
- The libvirt stuff is probably ok as it's a testing dependency.

**Comment by jkonecny12** [comment]
> * I think `npm` is needed to build, so it should go into the spec.

@KKoukiou what do you think? 

>     * `make` also seems suspicious to me. How come it's not pulled in via autotools?

The autotools is not dependency of the Anaconda. These tools are in the buildroot by default so you don't have to have these as dependencies in the spec file.

File: scripts/testing/install_dependencies.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 set -eu
 
-dnf install $@ rpm-build /usr/bin/xargs
+dnf install $@ make rpm-build /usr/bin/xargs npm libvirt virt-install
```
Comment (KKoukiou) [inline scripts/testing/install_dependencies.sh:31]: Isn't libvirt direct dependency of virt-install? 

If not, libvirt-daemon-kvm and libvirt-client are sufficient. Libvirt is a metapackage and installs a lot of uneeded things.

**Review by VladimirSlavik — APPROVED** [review]
Oh, sorry, that makes sense. The `spec` is second build, first we need the tarball contents.

File: scripts/testing/install_dependencies.sh (hunk 1 above)
Comment (jkonecny12) [reply scripts/testing/install_dependencies.sh:31]: I did not know about the libvirt-client, in that case yes, virt-install has a dependency on libvirt-client.

Comment (jkonecny12) [reply scripts/testing/install_dependencies.sh]: Fixed.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Removed libvirt, it's dragged in by virt-install.

**Comment by jkonecny12** [comment]
Rebased

**Review by poncovka — APPROVED** [review]

**Review by KKoukiou — APPROVED** [review]
the commit message just mentions npm, you add two more. But feel free to merge as is.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Rebased
* Change the commit message

**Pull request merged**